### PR TITLE
[WOR-156] exclude assigned managed apps

### PR DESCRIPTION
**Pull request by marctalbott**
Ticket: [WOR-156](https://broadworkbench.atlassian.net/browse/WOR-156)
* exclude all managed applications that have already been assigned to billing profiles in BPM
<!--
Hello, friend!
Remember to mention what sort of testing/verification you performed in the course of building this PR, i.e. checking functionality in the browser locally.
Also, so if a screen recording [1] and/or screenshots would be a helpful way to communicate context, please consider including some in your PR description.
Thanks!

[1] https://support.apple.com/guide/quicktime-player/record-your-screen-qtp97b08e666/mac
--!>

### CODE REVIEW

File: src/pages/billing/CreateAzureBillingProjectModal.js [hunk 1]
```diff
@@ -125,7 +126,13 @@ const CreateAzureBillingProjectModal = ({ onSuccess, onDismiss, billingProjectNa
         })
       ])]),
       h(IdContainer, [id => h(Fragment, [
-        h(FormLabel, { htmlFor: id, required: true }, ['Managed application']),
+        h(FormLabel, { htmlFor: id, required: true }, [
+          'Unassigned managed application',
+          h(InfoBox, { style: { marginLeft: '0.25rem' } }, [
```
Comment (marctalbott) [inline src/pages/billing/CreateAzureBillingProjectModal.js:131]: Should I add a unit test that asserts that this tooltip is rendered? Wasn't sure if it was necessary but happy to do so

Comment (cahrens) [reply src/pages/billing/CreateAzureBillingProjectModal.js:131]: I don't think it's necessary.

File: src/pages/billing/CreateAzureBillingProjectModal.test.js [hunk 2]
```diff
@@ -162,8 +162,8 @@ describe('CreateAzureBillingProjectModal', () => {
           listAzureManagedApplications: () => Promise.resolve(
             {
               managedApps: [
-                { applicationDeploymentName: 'testApp1', tenantId: 'fakeTenant1', subscriptionId: 'fakeSub1', managedResourceGroupId: 'fakeMrg1' },
-                { applicationDeploymentName: appName, tenantId: tenant, subscriptionId: subscription, managedResourceGroupId: mrg }
+                { applicationDeploymentName: 'testApp1', tenantId: 'fakeTenant1', subscriptionId: 'fakeSub1', managedResourceGroupId: 'fakeMrg1', assigned: true },
```
Comment (cahrens) [inline src/pages/billing/CreateAzureBillingProjectModal.test.js]: In practice, `assigned` would be false correct?

Comment (marctalbott) [reply src/pages/billing/CreateAzureBillingProjectModal.test.js]: Yeah, I don't know why I made this true

**Review by cahrens — APPROVED** [review]

**Review by blakery — APPROVED** [review]

**Pull request merged**